reana_db/models.py

```diff
@@ -137,27 +137,16 @@ def get_user_disk_usage(self, block_size=None):
         except Exception:
             return -1
 
-    def get_user_cpu_usage(self):
-        """Aggregate user CPU usage."""
-        try:
-            from .database import Session
+    def get_user_quota_by_type(self, resource_type):
```

Comment:
Minor: What do you think about omitting the word `user` from this function signature (and some of the ones above)? In the end, they always called from a user instance.

Reply:
I think it is a good idea, however, it would imply touching other repos, shall we create an issue for it? Maybe we can add it to this sprint, as a well-isolated task to prettify our current work.